bucket/fbide.json

```
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://fbide.freebasic.net",
+    "version": "0.4.6r4",
+    "license": "GPL-2.0-or-later",
+    "description": "#1 editor for FreeBASIC",
+    "url": "http://downloads.sourceforge.net/project/fbide/fbide%200.4/fbide%200.4.6%20r4/FBIde0.4.6r4.zip",
+    "hash": "sha1:326327c7082adae0af1df0281cde4af990b1e273",
+    "extract_dir": "FBIde0.4.6r4",
+    "depends": "freebasic",
+    "shortcuts": [
+        [
+            "fbide.exe",
+            "FBIde"
+        ]
+    ],
+    "pre_install": [
+        "$fbc = (versiondir 'freebasic' 'current') -replace '\\\\', '\\\\'",
+        "$pref = (Get-Content \"$dir\\IDE\\prefs_win32.ini\" -Raw) -replace 'fbc.exe', \"$fbc\\\\fbc.exe\"",
+        "$pref | Set-Content \"$dir\\IDE\\prefs_win32.ini\" -Encoding ASCII"
```

There is no need for pipe. This is slightly shorter and more readable

```suggestion
        "Set-Content \"$dir\\IDE\\prefs_win32.ini\" $pref -Encoding ASCII"
```